api/tests/identity-access-management/integration/infrastructure/repositories/user.repository.test.js

```diff
@@ -297,230 +297,280 @@ describe('Integration | Identity Access Management | Infrastructure | Repository
           expect(pagination).to.deep.equal(expectedPagination);
         });
       });
+      context('when default/contains request type is settled', function () {
+        context('when there are multiple users matching the same "first name" search pattern', function () {
+          beforeEach(function () {
+            databaseBuilder.factory.buildUser({ firstName: 'Son Gohan' });
+            databaseBuilder.factory.buildUser({ firstName: 'Son Goku' });
+            databaseBuilder.factory.buildUser({ firstName: 'Son Goten' });
+            databaseBuilder.factory.buildUser({ firstName: 'Vegeta' });
+            databaseBuilder.factory.buildUser({ firstName: 'Piccolo' });
+            return databaseBuilder.commit();
+          });
 
-      context('when there are multiple users matching the same "first name" search pattern', function () {
-        beforeEach(function () {
-          databaseBuilder.factory.buildUser({ firstName: 'Son Gohan' });
-          databaseBuilder.factory.buildUser({ firstName: 'Son Goku' });
-          databaseBuilder.factory.buildUser({ firstName: 'Son Goten' });
-          databaseBuilder.factory.buildUser({ firstName: 'Vegeta' });
-          databaseBuilder.factory.buildUser({ firstName: 'Piccolo' });
-          return databaseBuilder.commit();
-        });
-
-        it('should return only users matching "first name" if given in filter', async function () {
-          // given
-          const filter = { firstName: 'Go' };
-          const page = { number: 1, size: 10 };
-          const expectedPagination = { page: page.number, pageSize: page.size, pageCount: 1, rowCount: 3 };
-
-          // when
-          const { models: matchingUsers, pagination } = await userRepository.findPaginatedFiltered({ filter, page });
-
-          // then
-          expect(map(matchingUsers, 'firstName')).to.have.members(['Son Gohan', 'Son Goku', 'Son Goten']);
-          expect(pagination).to.deep.equal(expectedPagination);
-        });
-      });
+          it('should return only users matching "first name" if given in filter', async function () {
```

Comment:
```suggestion
          it('returns only users matching "first name" if given in filter', async function () {
```
Même chose pour les autres tests du fichier 😄